platform/android/src/run_loop.cpp

```diff
@@ -243,12 +252,28 @@ void RunLoop::stop() {
     });
 }
 
-void RunLoop::addWatch(int, Event, std::function<void(int, Event)>&&) {
-    throw std::runtime_error("Not implemented.");
+void RunLoop::addWatch(int fd, Event event, std::function<void(int, Event)>&& cb) {
```

Comment:
Where new addWatch and removeWatch is used?

Reply:
In platform/default HTTPFileSource::Impl::handleSocket().